### ISSUE TITLE
Group chat

### ISSUE THREAD

**Issue by deavmi** [Nutomic/ensichat#8]
This would be a great feature to add to allow conversations between groups of people.

**Comment by Nutomic** [comment]
Yep, this is planned, but needs some work ;)

**Comment by deavmi** [comment]
Is it hard to implement. Can a Bluetooth device connect to more than one device at a time?

**Comment by Nutomic** [comment]
Connectivity isn't the problem, as devices can connect to multiple neighbors, and messages are also forwarded. But it needs some protocol changes, and then quite some extra ui (create groups etc).

**Comment by deavmi** [comment]
Oh okay. I see.

**Comment by deavmi** [comment]
Also maybe groups can have a feature to allow symmetrical encryption for groups that want only certain users to join. Or maybe this can be done via a whitelist of fingerprints, yeah that sounds right.

**Comment by Nutomic** [comment]
This shouldn't be a problem, as long as only group members/admins can add new members?

**Comment by deavmi** [comment]
Yeah. I I also think there should then also be fully open groups.

**Comment by OffifialMITX** [comment]
What do you think about OpenPGP, which Kontalk is currently using?
